rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java

```diff
@@ -80,11 +80,11 @@
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public JNDIResourceModel getJndiReference();
+    public Iterable<JNDIResourceModel> getJndiReferences();
 
     /**
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public void setJndiReference(JNDIResourceModel jndiReference);
+    public void addJndiReference(JNDIResourceModel jndiReference);
```

Comment:
You can do that with edge name too. 



> On May 28, 2015, at 8:40 AM, Matej Briskar <notifications@github.com> wrote:
> 
> In rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java:
> 
> >  
> >      /**
> >       * Contains the jndi location for this resource.
> >       */
> >      @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
> > -    public void setJndiReference(JNDIResourceModel jndiReference);
> > +    public void addJndiReference(JNDIResourceModel jndiReference);
> @bradsdavis but there is another usage that it may bring. In case we would need to access all the global JNDIs, we would simply query the graph for all EJBJNDIs with enum set to GLOBAL.
> 
> —
> Reply to this email directly or view it on GitHub.
>